specification/apicenter/resource-manager/Microsoft.ApiCenter/preview/2023-07-01-preview/apicenter.json

```diff
@@ -0,0 +1,440 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Resource deleted successfully."
+          },
+          "204": {
+            "description": "Resource deleted successfully."
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ProvisioningState": {
+      "type": "string",
+      "description": "Provisioning state of the service.",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      },
+      "readOnly": true
+    },
+    "Service": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServiceProperties",
+          "description": "The properties of the service.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v5/managedidentity.json#/definitions/ManagedServiceIdentity",
+          "description": "The identity of the service."
+        }
+      },
+      "description": "The service entity.",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "ServiceCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Service"
+          },
+          "x-typespec-name": "Service[]",
+          "description": "The Service items on this page"
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "x-typespec-name": "Rest.ResourceLocation"
+        }
+      },
+      "description": "The response of a Service list operation.",
+      "required": [
+        "value"
+      ]
+    },
+    "ServiceProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The status of the last operation."
+        }
+      },
+      "description": "The properties of the service."
+    },
+    "ServiceUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ServiceProperties",
+          "x-ms-client-flatten": true
+        }
+      },
+      "description": "The service properties to be updated."
+    }
+  },
+  "parameters": {
+    "ServiceName": {
```

Comment:
@azaslonov The `serviceName` parameter defined here would result in a client-level parameter in SDK side, could you confirm that this is a client-level parameter or this is a method-level parameter shared by most operations?

If this is the latter one, please add the [x-ms-parameter-location](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-parameter-location) as `method`.

```json
"x-ms-parameter-location": "method" <<<<<<<<<<<<<<<<<<<<<<<<<<<
```

Reply:
@MaryGao, you're right. Fixed in this PR: https://github.com/Azure/azure-rest-api-specs/pull/25344